client/src/Tools/_framework/ToolPanels/CompoundActivityEditor.tsx

```diff
@@ -168,10 +170,14 @@ export function CompoundActivityEditor({
   // which should be given focus when drawers are closed
   const cardMenuRefs = useRef<HTMLButtonElement[]>([]);
 
-  const readOnly =
-    asViewer ||
-    (activity.assignmentInfo?.assignmentStatus ?? "Unassigned") !==
-      "Unassigned";
+  const isAssigned = activity.assignmentInfo
+    ? activity.assignmentInfo.assignmentStatus !== "Unassigned"
+    : false;
+
+  // Read only: cannot edit anything about this activity
+  // Read only structure: cannot add or remove sub-activities
```

Comment:
Is there a case where one wants `readOnly` but not `readOnlyStructure`? I would think if one is viewing it through a view or if the activity is assigned, one also shouldn't be able to add sub activities.